br/pkg/lightning/backend/backend_test.go

```diff
@@ -31,22 +32,25 @@ func Test(t *testing.T) {
 
 // FIXME: Cannot use the real SetUpTest/TearDownTest to set up the mock
 // otherwise the mock error will be ignored.
-
-func (s *backendSuite) setUpTest(c gomock.TestReporter) {
-	s.controller = gomock.NewController(c)
-	s.mockBackend = mock.NewMockBackend(s.controller)
-	s.backend = backend.MakeBackend(s.mockBackend)
-	s.ts = oracle.ComposeTS(time.Now().Unix()*1000, 0)
+func setUpTest(c gomock.TestReporter) *backendSuite {
+	controller := gomock.NewController(c)
+	mockBackend := mock.NewMockBackend(controller)
+	return &backendSuite{
+		controller:  controller,
+		mockBackend: mockBackend,
+		backend:     backend.MakeBackend(mockBackend),
+		ts:          oracle.ComposeTS(time.Now().Unix()*1000, 0),
+	}
 }
 
 func (s *backendSuite) tearDownTest() {
 	s.controller.Finish()
 }
 
-func (s *backendSuite) TestOpenCloseImportCleanUpEngine(c *C) {
-	s.setUpTest(c)
+func TestOpenCloseImportCleanUpEngine(t *testing.T) {
+	s := setUpTest(t)
 	defer s.tearDownTest()
-
+	setUpTest(t)
```

Comment:
Why do you call `setUpTest` twice in one test case?

Besides, please consider whether we can parallelize these tests.

Reply:
sorry ..is a mistake

Reply:
parallelize is okay